Server/mods/deathmatch/logic/CPlayer.h

```diff
@@ -276,6 +276,7 @@ class CPlayer : public CPed, public CClient
 
     CVehicle *                                  GetJackingVehicle           ( void )                        { return m_pJackingVehicle; }
     void                                        SetJackingVehicle           ( CVehicle * pVehicle );
+    bool                                        IsQuit                      ( void )                        { return bIsQuit; }

```

Comment:
You might want to name this method **IsLeavingServer**, but this is up for debate.